examples/embed-directive/embed-directive.go

```diff
@@ -0,0 +1,51 @@
+// Starting in Go version 1.16+, Go supports use of `//go:embed`
+// which is a directive to embed files and folders into
+// the application binary by the compiler.
+
+// Since Go does not have preprocessor or macros like C type of languages,
+// `pragmas/directives` are implemented by the compiler as comments `//`.
+// Using `//go:embed` tells the compiler to include arbitrary files/folders in binary.
```

Comment:
Comment lines are too long here -- please fold all to be 80 cols at most

Reply:
Will simplify.